### ISSUE TITLE
Building with rustc 1.17.0-nightly (fd182c401 2017-03-13)

### ISSUE THREAD

**Issue by frondeus** [reem/stainless#79]
Using fresh nightly rustc I tried to use stainless.
```
Compiling stainless v0.1.10
error[E0425]: cannot find function `tts_to_parser` in module `parse`
  --> /Users/frondeus/.cargo/registry/src/github.com-1ecc6299db9ec823/stainless-0.1.10/src/describe.rs:68:57
   |
68 |     let state: DescribeState = Parse::parse(&mut parse::tts_to_parser(cx.parse_sess(), tokens), (sp, &mut*cx, Some(name)));
   |                                                         ^^^^^^^^^^^^^ not found in `parse`

error: no method named `bump_and_get` found for type `&mut syntax::parse::parser::Parser<'_>` in the current scope
  --> /Users/frondeus/.cargo/registry/src/github.com-1ecc6299db9ec823/stainless-0.1.10/src/parse.rs:46:34
   |
46 |         let name = match (parser.bump_and_get(), parser.parse_ident().unwrap(), parser.bump_and_get()) {
   |                                  ^^^^^^^^^^^^

error: no method named `bump_and_get` found for type `&mut syntax::parse::parser::Parser<'_>` in the current scope
  --> /Users/frondeus/.cargo/registry/src/github.com-1ecc6299db9ec823/stainless-0.1.10/src/parse.rs:46:88
   |
46 |         let name = match (parser.bump_and_get(), parser.parse_ident().unwrap(), parser.bump_and_get()) {
   |                                                                                        ^^^^^^^^^^^^

error: no method named `bump_and_get` found for type `&mut syntax::parse::parser::Parser<'_>` in the current scope
   --> /Users/frondeus/.cargo/registry/src/github.com-1ecc6299db9ec823/stainless-0.1.10/src/parse.rs:183:23
    |
183 |     let real = parser.bump_and_get();
    |                       ^^^^^^^^^^^^

error: aborting due to 3 previous errors

error: Could not compile `stainless`.
```

I needed fresh nightly cause I couldn't compile clippy.
Should i degrade my rustc to highest common version (so I could use both clippy and stainless)?

**Comment by ujh** [comment]
Yes, I realised that as well. I will see what I can do about it.

**Comment by ujh** [comment]
Sorry for the delay! @pinkisemils fixed the issue and I just released 0.1.11. So that should resolve your issue.
